discovery_engine_flutter/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   xayn_discovery_engine:
     git:
       url: git@github.com:xaynetwork/xayn_discovery_engine_release.git
-      ref: main
+      ref: change_me_to_commit_ref
```

Comment:
How does that interact with local flutter builds/testing, non-release CI flutter testing?

Reply:
It doesn't interact because there is the override to the current directory